cmd/generate/runner.go

```diff
@@ -82,11 +82,22 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		ExtraLabels: nil,
 	}
 
+	if r.flag.Raw {
+		config, secret, err := gen.GenerateRaw(ctx, in)
+		if err != nil {
+			return microerror.Mask(err)
+		}
+		fmt.Println("---")
+		fmt.Printf(string(config) + "\n")
+		fmt.Println("---")
+		fmt.Printf(string(secret) + "\n")
+		return nil
+	}
+
 	configmap, secret, err := gen.Generate(ctx, in)
 	if err != nil {
 		return microerror.Mask(err)
 	}
```

Comment:
Wouldn't it be simpler to extract & print data from the CM and the Secret and leave internal/generator/service.go as it is?

Reply:
Sure, why not.